src/App/Fossa/Ficus/Analyze.hs

```diff
@@ -187,42 +210,50 @@ runFicus ficusConfig = do
       now <- getCurrentTime
       pure . formatTime defaultTimeLocale "%H:%M:%S.%3q" $ now
 
-    streamFicusOutput :: Handle -> IO (Maybe FicusSnippetScanResults)
-    streamFicusOutput handle =
+    streamFicusOutput :: Handle -> Maybe Handle -> IO (Maybe FicusSnippetScanResults)
+    streamFicusOutput handle maybeFile =
       Conduit.runConduit $
         CC.sourceHandle handle
           .| CC.decodeUtf8Lenient
           .| CC.linesUnbounded
+          .| CC.mapM
```

Comment:
[nit] This is [`iterMC`](https://hackage-content.haskell.org/package/conduit-1.3.6.1/docs/Conduit.html#v:iterMC) I think.